recipes/pedantic/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "pedantic" %}
+{% set version = "1.14.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/LostInDarkMath/pedantic-python-decorators/archive/refs/tags/20230913095009-ca2b278.tar.gz
+  sha256: 169cbeb2382ad999a9c6cf203304b83a003e404f31cd51ffcb2c09e5b3dcc501
```

Comment:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pedantic-{{ version }}.tar.gz
  sha256: 00359efc587023fe8c767f054eee6ae49cf6c618e230b897f49b34dc936e65c7
```

Reply:
This is how I had it but it built an empty package, we can give it another try